pkgs/by-name/co/cosmic-screenshot/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rust
+, rustPlatform
+, just
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-screenshot";
+  version = "unstable-2023-11-08";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = pname;
+    rev = "b413a7128ddcdfb3c63e84bdade5c5b90b163a9a";
+    hash = "sha256-SDxBBhmnqNDX95Rb7QiI46sAxrfodB5tSq8AbXAU480=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
I don't think we need to vendor the lock file here, do we?

Reply:
We do since rustPlatform doesn't support git cargo dependencies yet (someone should probably get on that, though)

Reply:
but wouldn't we have a hash here if git dependencies would be used?

Reply:
Fixed